devel/qt5-qtcreator/Portfile

```diff
@@ -49,7 +49,7 @@ license_noconflict  openssl
 
 if { ${subport} eq ${name}  } {
 
-    # Requires Xcode
+    # Requires actool
```

Comment:
this does not make things any more clear to me... am I supposed to know that when `actool` is required that this means a full Xcode installation is needed?

Reply:
Possibly, though maybe based on https://trac.macports.org/ticket/59348 what I should say is something like:

```
# Requires `actool` (not included with command line tools)
# Must request full Xcode so that `actool` is usable
# See: https://trac.macports.org/ticket/59280
# See: https://trac.macports.org/ticket/59348
use_xcode yes
```

Although I might be inclined just to use the actual error instead. Does it say enough?
```
# xcode-select: error: tool 'actool' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
use_xcode yes
```

Reply:
yes, that is certainly more clear than the original and the initially proposed change